Graphs/Table/index.js

```diff
@@ -331,6 +337,41 @@ const TableGraph = (props) => {
         );
     }
 
+    const hoverContent = (cellData, columnIndex, rowIndex) => {
+        return <div key={`tooltip_${cellData}_${columnIndex}_${rowIndex}`}>
+            {cellData} &nbsp;
+        <CopyToClipboard text={cellData ? cellData.toString() : ''}>
+                <button style={{ background: '#000', padding: 1 }} title="copy">
+                    <FaRegClipboard size={10} color="#fff" />
+                </button>
+            </CopyToClipboard>
+        </div>
+    }
+
+    const renderTooltip = (cellData, columnIndex, rowIndex) => {
+        const place = rowIndex === currentStartIndex ? 'bottom' : 'top';
+        const top = rowIndex === currentStartIndex ? (rowHeight * (rowIndex + 1)) - 20 : (rowHeight * rowIndex) - 15;
+
+        return <ReactTooltip
+            id={`tooltip_${cellData}_${columnIndex}_${rowIndex}`}
+            place={place}
+            delayUpdate={1000}
+            overridePosition={() => ({ left: (graphWidth / columns.length) * columnIndex - 3 * cellData.length, top })}
+            getContent={[() => hoverContent(cellData, columnIndex, rowIndex)]}
+        />;
+    }
+
+    const cellRendererData = (cellData, columnIndex, rowIndex, column) => {
+        const data = (!!column.totalCharacters && !!cellData && cellData.length > column.totalCharacters) ? cellData.slice(0, column.totalCharacters) + '...' : cellData;
+
+        return (
+            <div>
+                <p data-tip='' data-for={`tooltip_${cellData}_${columnIndex}_${rowIndex}`} delayUpdate={1000}>{data}</p>
```

Comment:
do we need this when tooltip is not applicable? also, is delayUpdate required within paragraph tag?
can we change this as,
```
return !!column.tooltip ?
    <React.Fragment>
        <p data-tip='' data-for={`tooltip_${cellData}_${columnIndex}_${rowIndex}`}>{data}</p>
        {renderTooltip(cellData, columnIndex, rowIndex)}
    </React.Fragment> : data;
```